src/services/db/RepoService.ts

```diff
@@ -0,0 +1,245 @@
+    return ReviewApi.getCommitDiff(siteName, base, head)
+  }
+
+  createPullRequest(
+    siteName: string,
+    title: string,
+    description?: string,
+    base?: string,
+    head?: string
+  ) {
+    return ReviewApi.createPullRequest(siteName, title, description, base, head)
+  }
+
+  getPullRequest(siteName: string, pullRequestNumber: number) {
+    return ReviewApi.getPullRequest(siteName, pullRequestNumber)
+  }
+
+  getBlob(repo: string, path: string, ref: string) {
+    return ReviewApi.getBlob(repo, path, ref)
+  }
+
+  updatePullRequest(
+    siteName: string,
+    pullRequestNumber: number,
+    title: string,
+    description?: string
+  ) {
+    return ReviewApi.updatePullRequest(
+      siteName,
+      pullRequestNumber,
+      title,
+      description
+    )
+  }
+
+  closeReviewRequest(siteName: string, pullRequestNumber: number) {
+    return ReviewApi.closeReviewRequest(siteName, pullRequestNumber)
+  }
+
+  mergePullRequest(siteName: string, pullRequestNumber: number) {
+    return ReviewApi.mergePullRequest(siteName, pullRequestNumber)
+  }
+
+  approvePullRequest(siteName: string, pullRequestNumber: number) {
+    return ReviewApi.approvePullRequest(siteName, pullRequestNumber)
+  }
+
+  async getComments(siteName: string, pullRequestNumber: number) {
+    return ReviewApi.getComments(siteName, pullRequestNumber)
+  }
+
+  async createComment(
+    siteName: string,
+    pullRequestNumber: number,
+    user: string,
+    message: string
+  ) {
+    return ReviewApi.createComment(siteName, pullRequestNumber, user, message)
+  }
+
+  getFilePath({ siteName, fileName, directoryName }: any): any {
+    return super.getFilePath({
+      siteName,
+      fileName,
+      directoryName,
+    })
+  }
+
+  getBlobPath({ siteName, fileSha }: any): any {
+    return super.getBlobPath({ siteName, fileSha })
+  }
+
+  getFolderPath({ siteName, directoryName }: any) {
+    return super.getFolderPath({ siteName, directoryName })
+  }
+
+  async create(
+    sessionData: any,
+    { content, fileName, directoryName, isMedia = false }: any
+  ): Promise<any> {
+    return await super.create(sessionData, {
+      content,
+      fileName,
+      directoryName,
+      isMedia,
+    })
+  }
+
+  async read(
+    sessionData: UserWithSiteSessionData,
+    { fileName, directoryName }: { fileName: string; directoryName?: string }
+  ): Promise<GitFile> {
+    if (this.isRepoWhitelisted(sessionData.siteName)) {
+      logger.info("Reading file from local Git file system")
+      const filePath = directoryName ? `${directoryName}/${fileName}` : fileName
+      const result = await this.gitFileSystemService.read(
+        sessionData.siteName,
+        filePath
+      )
+
+      if (result.isErr()) {
+        throw result.error
+      }
+
+      return result.value
+    }
+
+    return await super.read(sessionData, {
+      fileName,
+      directoryName,
+    })
+  }
+
+  // TODO: This is no longer used, remove it
```

Comment:
I was not able to find any usages of this function anywhere, and I think it was removed in PR #740 without the function being removed. I added this todo here in case I missed out something + felt that it would be out of scope of this PR.